runtime/fastly/builtins/fastly.cpp

```diff
@@ -43,6 +43,9 @@ JS::PersistentRooted<JSObject *> Fastly::env;
 JS::PersistentRooted<JSObject *> Fastly::baseURL;
 JS::PersistentRooted<JSString *> Fastly::defaultBackend;
 bool Fastly::allowDynamicBackends = false;
+host_api::BackendConfig Fastly::defaultDynamicBackendConfig = host_api::BackendConfig{
+    std::nullopt, std::nullopt, std::nullopt, std::nullopt, std::nullopt, std::nullopt,
+    std::nullopt, std::nullopt, std::nullopt, std::nullopt, std::nullopt, std::nullopt};
```

Comment:
Same comment here, you don't need to explicitly initialize these.

Reply:
```suggestion
host_api::BackendConfig Fastly::defaultDynamicBackendConfig;
```